src/descriptors/power_system_structs.json

```
@@ -94,36 +94,30 @@
       "supertype": "OperationalCost"
     },
     {
-      "struct_name": "MultiStartCost",
-      "docstring": "Data Structure Operational Cost Data which includes fixed, variable cost, multiple start up cost and stop costs.",
+      "struct_name": "MarketBidCost",
+      "docstring": "Data Structure Operational Cost to reflect market bids of energy and ancilliary services.\nCompatible with most US Market bidding mechanisms",
       "fields": [
         {
-          "name": "variable",
+          "name": "energy_bid",
           "comment": "variable cost",
           "null_value": "VariableCost((0.0, 0.0))",
           "data_type": "VariableCost"
         },
         {
-          "name": "no_load",
+          "name": "no_load_cost",
```

```suggestion
          "name": "no_load",
```
I'd prefer to keep it as `no_load` for consistency with the other names that don't have `_cost` (e.g. `start_up`, `shut_down`, `fixed`, ...)